app/Http/Requests/StandCreateRequest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StandCreateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'identifier' => 'required|string',
```

Comment:
If you're able to give https://github.com/VATSIM-UK/uk-controller-api/pull/441 a quick review, we can add `assignment_priority` here as a positive integer?

Reply:
Reviewed, can you merge before I push my changes?

Reply:
Also, is it optional?

Reply:
I've merged now - the default is 100 on that field, so it is optional in the DB.

Reply:
It didn't allow a NULL, so I added a default on the controller model create. https://github.com/VATSIM-UK/uk-controller-api/pull/443/commits/cbcff762530e2743e534cae9739722c5d046bee0#diff-ea33c1e1edc02e130249c2cb106990430cc2389c4c0fc20c6b8d10e33b2708f1R108